CHANGELOG

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 
+## 0.1.2
```

Comment:
This line should say "develop" or "Pre-Release" or similar. Let's NOT give version numbers until we are ready to publish said version to master and tag it.